docs/UserGuide.md

```diff
@@ -121,19 +122,20 @@ Format: `list`
 
 Edits an existing patient in the address book.
 
-Format: `edit INDEX [id/PATIENT_HOSPITAL_ID] [n/NAME] [p/PREFERRED_NAME] [f/FOOD_PREFERENCE] [c/FAMILY_CONDITION]
-[h/HOBBY] [t/TAG]…​`
+Format: `edit INDEX [id/PATIENT_HOSPITAL_ID] [n/NAME] [p/PREFERRED_NAME] [f/FOOD_PREFERENCE]+ [c/FAMILY_CONDITION]+
```

Comment:
I realised that this command can edit 0 or more food preference, family condition and hobby which is different from Add command. So it should be [f/ FOOD_PREFERENCE]... [c/FAMILY_CONDITION]... [h/HOBBY]...

Reply:
yep nice catch! have updated this!